sound/attributions.txt

```diff
@@ -17,4 +17,5 @@ captured by the BBC
 "Gunfire & Guns: Revolver, Chambers spinning.": http://bbcsfx.acropolis.org.uk/assets/07027165.wav
 bbc.co.uk - (c) copyright 2018 BBC
 
-kiss sound: https://freesound.org/people/stereostereo/sounds/117355/
\ No newline at end of file
+kiss sound: https://freesound.org/people/stereostereo/sounds/117355/
+champagne_pop.ogg is credited to ultradust on freesound https://freesound.org/people/ultradust/sounds/166923/
```

Comment:
```suggestion
champagne_pop.ogg is credited to ultradust on freesound https://freesound.org/people/ultradust/sounds/166923/

```

New line prevents blames from getting worse, just like this diff shows with the kiss sound.